boards/xtensa/intel_s1000_crb/doc/index.rst

```diff
@@ -173,6 +173,25 @@ application.
 Refer to :ref:`build_an_application` and :ref:`application_run` for
 more details.
 
+Downloading binary image
+========================
+
+A Linux host connected to the SPI interface of the ``intel_s1000_crb`` board
+can download a zephyr binary to RAM and execute the image.
+
+.. code-block:: console
+
+   cd <app-dir>/build
+   sudo python3 $ZEPHYR_BASE/boards/xtensa/intel_s1000_crb/support/download.py \
+   zephyr/zephyr.bin
+
+The script depends on a few python modules. These dependencies can be installed
+on the Linux host using the command below.
+
+.. code-block:: console
+
+   sudo pip3 install pyyaml python-periphery hashlib bitstruct
```

Comment:
We've been recommending folks use ``--user`` rather than using ``sudo`` to install stuff with pip:
```
pip3 install --user pyyaml python-periphery hashlib bitstruct
```

Reply:
> We've been recommending folks use `--user` rather than using `sudo` to install stuff with pip:
> 
> ```
> pip3 install --user pyyaml python-periphery hashlib bitstruct
> ```

Changed.

Reply:
The download.py would probably not work at all since it is run as root, so these packages would not be visible to the script.

Reply:
> The download.py would probably not work at all since it is run as root, so these packages would not be visible to the script.

suggested solution? go back to installing the packages via `sudo pip3`?